tflitehub/inception_v4_uint8_test.py

```diff
@@ -0,0 +1,32 @@
+# RUN: %PYTHON %s
+
+import absl.testing
+import imagenet_test_data
+import numpy
+import test_util
+
+# Source https://tfhub.dev/tensorflow/lite-model/inception_v4_quant/1/default/1
```

Comment:
Should we perhaps use inception_v4_quant in the name of the test file to make tracking back to tfhub easier?

Reply:
It's useful to differentiate between UINT8 and INT8 models, which the tfhub naming unfortunately doesn't do. For example, that accuracy regression that we just ran into only occurred in INT8 models and it was really easy to tell by looking at the test name.